docs/examples/deep_corrector.md

```diff
@@ -113,7 +113,7 @@ This model should perform better on the given inputs compared to the previous on
 If you run a job with `--mode serve` flag, FloydHub will run the `app.py` file in your project and attach it to a dynamic service endpoint:
 
 ```bash
-floyd run --mode serve --env tensorflow-1.5 --data floydhub/deep-text-corrector/23/:input
+floyd run --mode serve --env tensorflow-1.5 --data <REPLACE_WITH_JOB_OUTPUT_NAME>:input
```

Comment:
what is `job_output_name` in the new, non-output workflow? Does this still make sense. Wouldn't it just be the job name?

Reply:
Good call! I'll fix it